components/automate-ui/src/app/services/profiles/available-profiles.service.ts

```diff
@@ -33,7 +33,10 @@ export class AvailableProfilesService {
     const params = new HttpParams().set('owner', this.chefSessionService.username);
     const headers = new HttpHeaders({'Content-Type': 'application/json'});
 
-    return this.httpClient.post(url, body, {params, headers});
+    return this.httpClient.post(url, body, {params, headers}).pipe(
+      catchError((err) => {
+        return throwError(err);
```

Comment:
This looks suspicious. 🤔 What happens without this? Would the caller _not_ get an error thrown?